src/assessment.js

```diff
@@ -49,5 +50,6 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  console.log(this)
 };
+  fire.call(deathStar);
```

Comment:
Using `call` will work, but not as you've written it. Also, why is it outside the `fireOnAlderaan` function?